src/clients/mocks/MockSpokePoolClient.ts

```diff
@@ -63,15 +64,15 @@ export class MockSpokePoolClient extends SpokePoolClient {
     this.realizedLpFeePctOverride = false;
   }
 
-  async computeRealizedLpFeePct(depositEvent: FundsDepositedEvent) {
+  async computeRealizedLpFeePct(depositEvent: FundsDepositedEvent | V3FundsDepositedEvent) {
```

Comment:
Yes, but if I remove it now it requires additional changes to be made in other files. I was trying to limit the scope of this one for the time being so that we don't have to make wholesale test changes, so I'll leave it as-is for now and will circle back after the follow-on PRs are merged.

Reply:
Fixed: https://github.com/across-protocol/sdk-v2/pull/594/commits/b4d3e0c1b526845e14783f39682f3907750ccdba